docs/integration/local_user.md

```diff
@@ -0,0 +1,54 @@
+# Map ORCID IDs to Local User Accounts
+
+The Data Transfer System (DTS) uses [ORCID IDs](https://info.orcid.org/what-is-orcid/)
```

Comment:
if you want to be cool like the cool kids at ORCID, you can write it as "ORCID iD". 😎